tests/metadata_stores/conftest.py

```diff
@@ -15,6 +26,42 @@
 
 assert HashAlgorithmCases is not None  # ensure the import is not removed
 
+logger = logging.getLogger(__name__)
+
+# Force pytest-postgresql to use a short socket directory to avoid hitting the
+# 103-character Unix socket limit enforced by PostgreSQL on macOS.
+_PG_SOCKET_DIR = Path("/tmp/metaxy-pg")
+_PG_SOCKET_DIR.mkdir(parents=True, exist_ok=True)
+
+postgresql_proc = factories.postgresql_proc(unixsocketdir=str(_PG_SOCKET_DIR))
+
+
+def _find_free_port() -> int:
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.bind(("127.0.0.1", 0))
+        s.listen(1)
+        port = s.getsockname()[1]
+    return port
+
+
+def _format_pg_connection_string(
```

Comment:
Curious if there is one available in psycopg2 or ibis, but I guess it's not very important

Reply:
removed